docs/querying/segmentmetadataquery.md

```diff
@@ -194,15 +195,20 @@ null if the aggregators are unknown or unmergeable (if merging is enabled).
 * `rollup` in the result is true/false/null.
 * When merging is enabled, if some are rollup, others are not, result is null.
 
-## lenientAggregatorMerge
+### aggregatorMergeStrategy
 
 Conflicts between aggregator metadata across segments can occur if some segments have unknown aggregators, or if
-two segments use incompatible aggregators for the same column (e.g. longSum changed to doubleSum).
+two segments use incompatible aggregators for the same column (e.g. `longSum` changed to `doubleSum`). The following
+aggregator merge strategies are supported:
 
-Aggregators can be merged strictly (the default) or leniently. With strict merging, if there are any segments
-with unknown aggregators, or any conflicts of any kind, the merged aggregators list will be `null`. With lenient
-merging, segments with unknown aggregators will be ignored, and conflicts between aggregators will only null out
-the aggregator for that particular column.
+- `strict`:  If there are any segments with unknown aggregators, or any conflicts of any kind, the merged aggregators
+  list will be `null`.
+- `lenient`:  Segments with unknown aggregators will be ignored, and conflicts between aggregators will only null out
```

Comment:
```suggestion
- `lenient`:  Druid ignores segments with unknown aggregators. Conflicts between aggregators send the aggregator for that particular column to null.
```
I am not sure if this changes the original meaning but consider avoiding future tense and passive voice.